src/ocamlorg_package/lib/opam_repository.mli

```diff
@@ -7,8 +7,9 @@ val pull : unit -> unit Lwt.t
 val last_commit : unit -> string Lwt.t
 (** Get the latest commit of the opam repository. *)
 
-val list_packages : unit -> string list
```

Comment:
I think we can keep this function, it seems useful, even if we have a function `list_packages_and_versions`.

Reply:
I removed it because it's not used anymore. I don't believe it can be ever useful because we want to use the cached and processed list of packages we already keep in memory. These private functions are only used to refresh this internal cache and won't be used outside of `Ocamlorg_package` and `Packages_stats`.
I've added it back but suggest reverting.